ingest/cell_metadata.py

```diff
@@ -23,11 +23,11 @@ def __init__(self, file_path, file_id: str, study_accession: str, *args, **kwarg
         # unique values for group-based annotations
         self.unique_values = []
         self.cell_names = []
-        self.annotation_type = ["group", "numeric"]
+        self.annotation_type = ['group', 'numeric']
         self.top_level_doc = self.create_documents(file_id, study_accession)
         self.data_subcollection = self.create_subdocuments()
-        self.errors = defaultdict(list)
-        self.ontology = defaultdict(lambda: defaultdict(set))
+        self.errors = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
```

Comment:
I would add a comment here for sure. I believe you're trying to create a dictionary of dictionaries?

Reply:
added comment to explain in `29aa7e3`